### ISSUE TITLE
[BUG] - Flight submission and logbook viewing issues (w/phpvms7)

### ISSUE THREAD

**Issue by jordanhavard** [invernyx/smartcars-3-bugs#160]
### Describe the bug

I believe the app is expecting the `pirep_id` to be a number of type `int` however with phpvms7 this isn't the case. I think this is the reason why PIREPs are failing to submit and the logbook is not able to be viewed

### How do you reproduce this bug?

1. Try submit a flight in PHPVMS7 with a pirepID something other than an INT

### Expected behavior

Not sure why this is type checked as the app doesn't seem to do anything with it. 
Ideally UUID, String or INT should be allowed types

### Operating system

Win 10 Pro 10.0.19044

### Community airline

vAAirlines

### smartCARS Version

0.8.1.2

### Plugins installed

all except simbrief

### Additional context

Offending lines of code

```
if ("number" != typeof i.data.pirepID) throw new C(i.data.message, 200, i.data, "flights/complete");
```

**Comment by jordanhavard** [comment]
A tempoary fix for those experiencing issues is to change this line in `complete.php` 
`echo(json_encode(array('pirepID' => $pirepID)));`
to
`echo(json_encode(array('pirepID' => 1)));`

**Comment by turbofandude** [comment]
@GenericNerd and @bgiorgio0506 please review/fix

**Comment by bgiorgio0506** [comment]
will be fixed next update

**Comment by turbofandude** [comment]
Fixed in 0.8.3+
